HQSmokeTests/testPages/users/group_page.py

```diff
@@ -82,3 +82,19 @@ def cleanup_group(self):
         self.wait_to_click(self.confirm_delete)
         assert self.is_visible_and_displayed(self.delete_success_message), "Group deletion not successful"
         print("Clean up added group")
+
+    def delete_test_groups(self):
+        list_profile = self.driver.find_elements(By.XPATH,"//td//a[contains(text(),'group_')]")
```

Comment:
would it make sense to move xpath` self.driver.find_elements(By.XPATH,"//td//a[contains(text(),'group_')]")` and store it in a variable?

Reply:
Same for role

Reply:
I tried to make a variable for this one but we are using find_elements instead find_element so it kind of failed. That's why I wrote it this way.

Reply:
Probably because brackets aren't required there?

Reply:
which brackets? the ones in the variable declaration?

Reply:
I tried both with brackets and without brackets and both times I got 
![nobrackets](https://user-images.githubusercontent.com/24750350/179489371-dd719195-f004-4131-94d2-77d7ee03dc48.jpg)